koku/masu/openapi.json

```diff
@@ -812,7 +812,92 @@
         ]
       }
     },
-    "/gcp_invoice_monthly_cost/": {
+    "/db-performance/db-version/": {
+        "get": {
+          "summary": "Returns a table showing the PostgreSQL database software version.",
+          "operationId": "dbPerformanceDbVersion",
+          "description": "Returns a HTML document showing the database version.",
+          "responses": {
+            "200": {
+              "description": "Returns a HTML document showing the database version.",
+              "content": "text/html"
+            }
+          },
+          "tags": [
+            "db-performance",
+            "db-version"
+          ]
+        }
+      },
+      "/db-performance/db-settings/": {
+        "get": {
+          "summary": "Returns a table showing the PostgreSQL database software version.",
+          "operationId": "dbPerformanceDbSettings",
+          "description": "Returns a HTML document showing the database software settings.",
+          "responses": {
+            "200": {
+              "description": "Returns a HTML document showing the database software settings.",
+              "content": "text/html"
```

Comment:
I added this because people kept asking about that file.
But: If these endpoints are returning HTML instead of JSON, do they really belong in openapi.json?